pycloudlib/azure/cloud.py

```diff
@@ -793,7 +804,10 @@ def snapshot(self, instance):
                     "id": instance.id
                 },
                 "tags": {
-                    "name": self.tag
+                    "name": self.tag,
+                    "id": "pycloudlib-{}-{}".format(
+                        instance.offer, instance.sku
+                    )
```

Comment:
Can we add an instance_id property to Azure instance and name this tag key  "src-img-id" and make it the same format as the original launched image_id?


```suggestion
                    "src-image-id": instance.instance_id
```


This would involve addiing the following to azure/instance.py
```diff
diff --git a/pycloudlib/azure/instance.py b/pycloudlib/azure/instance.py
index 56bd218..9f75b3e 100644
--- a/pycloudlib/azure/instance.py
+++ b/pycloudlib/azure/instance.py
@@ -32,6 +32,16 @@ class AzureInstance(BaseInstance):
         """Wait for instance stop."""
         raise NotImplementedError
 
+    @property
+    def image_id(self):
+        """Return the image_id from which this instance was created."""
+        storage_profile = self._instance['vm'].as_dict()['storage_profile']
+        image_ref = storage_profile['image_reference']
+
+        return ":".join(
+            [image_ref['publisher'].lower(), image_ref['offer'],
+             image_ref['sku'], image_ref['version']])
+
     @property
     def ip(self):
         """Return IP address of instance."""
```